.github/workflows/manifest_updateVersions.createPR.yml

```diff
@@ -0,0 +1,92 @@
+name: Update Versions and Create PR (manifest)
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Release Version (x.x.x)'
+        required: true
+      zowe_version:
+        description: "Zowe Version Tag"
+        default: zowe-v2-lts
+        required: true
+        type: choice
+        options:
+        - zowe-v1-lts
+        - zowe-v2-lts
+
+jobs:
+  update_versions_and_create_pr:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Set Default Branch Environment Variable
+        run: |
+          if [[ "${{ github.event.inputs.zowe_version }}" == "zowe-v1-lts" ]]; then
+            echo "defaultBranch=v1.x/staging" >> $GITHUB_ENV
+          else
+            echo "defaultBranch=v2.x/staging" >> $GITHUB_ENV
+          fi
+
+      - name: Checkout zowe-install-packaging
+        uses: actions/checkout@v4
+        with:
+          repository: 'zowe/zowe-install-packaging'
+          ref: ${{ env.defaultBranch }}
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: lts/*
+
+      - name: Create Branch
+        run: |
+          branch_name="v${{ github.event.inputs.version }}/rc"
+          git checkout -b "$branch_name"
+
+      - name: Get and Change Component Versions
+        id: get_versions
+        run: |
+          declare -A repoAndComponent
+          repoAndComponent[imperative]=imperative
+          repoAndComponent[cli]=zowe-cli
+          repoAndComponent[cics-for-zowe-cli]=zowe-cli-cics-plugin
+          repoAndComponent[db2-for-zowe-cli]=zowe-cli-db2-plugin
+          repoAndComponent[perf-timing]=perf-timing
+          repoAndComponent[mq-for-zowe-cli]=zowe-cli-mq-plugin
+          repoAndComponent[zos-ftp-for-zowe-cli]=zowe-cli-ftp-plugin
+
+          zowe_version=${{ github.event.inputs.zowe_version }}
```

Comment:
If the Zowe version is zowe-v1-lts, we should add [`secure-credential-store-for-zowe-cli`](https://github.com/zowe/zowe-cli-scs-plugin) to the "repoAndComponent" list.